### DISCUSSION TITLE
How to set music or chat applications to match respective faders?

### DISCUSSION THREAD

**Discussion by douglasrcjames** [Adelenade/GoXlr-Macos#23] [Q&A]
Hi, I am looking to match up my Spotify to the Music fader and Zoom, Discord, etc to the Chat fader so when I lower the Music fader, the Music will decrease in volume. On Windows I know how to go into sound settings and match the app up with the proper input driver for that fader, but not sure how to do that in this case. Any ideas?

**Comment by startix86** [comment]
Looking around on the Mac side myself as I just discovered this awesome project. I still need to test it but the features and screenshots of SOUNDSOURCE by Rogue Amoeba looks to be a way to route sources further similar to Windows - to your GoXLR virtual channels (Music, Game, System, etc). Will update after I give it a whirl.
